internal/cmd/pipeline/command_update.go

```diff
@@ -32,12 +32,12 @@ func (c *command) newUpdateCommand(enableSourceCode bool) *cobra.Command {
 				Code: `confluent pipeline update pipe-12345 --activation-privilege false`,
 			},
 			examples.Example{
-				Text: `Request to update Stream Designer pipeline "pipe-12345", with new ksql cluster Id.`,
+				Text: `Update Stream Designer pipeline "pipe-12345" with KSQL cluster ID "lksqlc-123456".`,
 				Code: `confluent pipeline update pipe-12345 --ksql-cluster lksqlc-123456.`,
 			},
 			examples.Example{
-				Text: `Request to update Stream Designer pipeline "pipe-12345", with new schema registry cluster Id.`,
-				Code: `confluent pipeline update pipe-12345 --update-schema-registry`,
+				Text: `Update Stream Designer pipeline "pipe-12345" with new schema registry cluster Id.`,
+				Code: `confluent pipeline update pipe-12345 --update-schema-registry true`,
```

Comment:
This is redundant. The existence of this flag implies `true`.
```suggestion
				Code: `confluent pipeline update pipe-12345 --update-schema-registry`,
```